docs/docusaurus/docs/core/introduction/try_gx.md

```diff
@@ -1,7 +1,153 @@
+
+   ```python title="Python output" name="docs/docusaurus/docs/core/introduction/try_gx.py output1"
+   ```
+
+5. Create an Expectation that will fail.
+
+   Not all Expectations will pass successfully when validated.  If the data does not match the state described in an Expectation, the Expectation will fail.
+
+   A failed Expectation lets you know that you have an issue to look into: it could be something wrong with the data itself, such as missing or incorrect values.  Or it could be a misunderstanding about the data, in which case the Expectation itself should be updated.
+
+   This code creates an Expectation that will fail because it follows the incorrect assumption that taxicabs can only seat a maximum of three passengers:
+
+   ```python title="Python input" name="docs/docusaurus/docs/core/introduction/try_gx.py validate and view failed results"
+   ```
+
+   You can see that the results of a failed Expectation are very different, and include metrics to help you assess the severity of the issue you've uncovered:
+
+   ```python title="Python output" name="docs/docusaurus/docs/core/introduction/try_gx.py failed output"
+   ```
+
+   By default, only a portion of the failed values and record indexes will be included in the Validation Results to prevent the report from being too bloated to easily review.  However, the failed counts and percentages will correspond to all the failed records in the validated data.
+
+6. Optional.  Experiment with other Expectations.
+
+   The {GxData.product_name} library comes with numerous Expectations you can apply to your data.  You can browse the available Expectations through the [Expectations Gallery](https://greatexpectations.io/expectations).  Each entry in the Expectations Gallery links to a page with comprehensive information about the Expectation in question.  You can reference these pages to find details on how the Expectation functions and what parameters need to be provided when implementing it. 
+
+   The Expectations Gallery also includes search and filter functionality to facilitate finding appropriate Expectations for your use case.  For instance, this guide uses pandas as the backend for reading data.  Therefore, if you want to try additional Expectations you should [filter the Expectation Gallery to find Expectations that support pandas](https://greatexpectations.io/expectations/?viewType=Summary&filterType=Backend+support&showFilters=true&subFilterValues=pandas).
+
+</TabItem>
+
+<TabItem value="sample_code" label="Sample code">
+
+```python title="Full example script" name="docs/docusaurus/docs/core/introduction/try_gx.py full example script"
+```
+
+</TabItem>
+
+</Tabs>
+
+## Next steps
+
+You can learn more about GX by browsing through [Community resources](/core/introduction/community_resources.md) such as the GX Discourse forum, public Slack channel, and GitHub repo.
+
+### Try GX Cloud
+
+<GxCloudAdvert/>
+
+### Dive in with GX and Python
```

Comment:
Remove the heading and move this content so it follows the introductory statement.